### PR TITLE
Added documentation for BP volume ops

### PR DESCRIPTION
**Pull request by nishith-vihar**
Added documentation for BP volume ops

Fixes: #394
Signed-off-by: Nishith Vihar Sakinala <nsakinal@redhat.com>

### CODE REVIEW

File: docs/BP/Ops/volume_ops.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
+# Volume Ops
+
+Volume Ops contains all the functions required to carry out the volume related operatins. Given below are all the details about all the functions implemented in the Volume Ops library:
```
Comment (aujjwal-redhat) [inline docs/BP/Ops/volume_ops.md:3]: operatins => operations

**Review by aujjwal-redhat — CHANGES_REQUESTED** [review]

File: docs/BP/Ops/volume_ops.md (hunk 1 above)
Comment (nishith-vihar) [reply docs/BP/Ops/volume_ops.md]: Ack

File: docs/BP/Ops/volume_ops.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
+# Volume Ops
+
+Volume Ops contains all the functions required to carry out the volume related operations. Given below are all the details about all the functions implemented in the Volume Ops library:
+
+1) [**volume_mount**](https://github.com/srijan-sivakumar/redant/blob/d95bd1a1da8d16131bedf37f8d2ebf8f5c3259f9/common/ops/gluster_ops/volume_ops.py#L16)<br>
```
Comment (nik-redhat) [inline docs/BP/Ops/volume_ops.md:5]: Better to use the relative path here and all other places below.

Comment (nik-redhat) [reply docs/BP/Ops/volume_ops.md:5]: And in other docs as well.

Comment (nishith-vihar) [reply docs/BP/Ops/volume_ops.md:5]: I am not sure if we can use relative paths to point to a particular 'line' in a file.

Comment (nik-redhat) [reply docs/BP/Ops/volume_ops.md:5]: Why? Have you tried?

Comment (nishith-vihar) [reply docs/BP/Ops/volume_ops.md:5]: My bad. I got how to do that. I will make the changes

**Review by srijan-sivakumar — APPROVED** [review]

**Pull request merged**